eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/Utxo.hs

```diff
@@ -387,7 +387,7 @@ genNextDelta
         -- increase when we add the delta to the tx?
         draftSize =
           sum
-            [ 1100 :: Integer -- safety net in case the coin or a list prefix rolls over into a
+            [ 11000 :: Integer -- safety net in case the coin or a list prefix rolls over into a
```

Comment:
I have experimented with decreasing generated minFeeA and minFeeB - if we dramatically decrease it here in [Constants](https://github.com/IntersectMBO/cardano-ledger/blob/master/eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Constants.hs#L89) , as  I did [in this commit](https://github.com/IntersectMBO/cardano-ledger/commit/01f01f68be632f9dac8e186d9520b4ab42006a46), tests pass reliably. 
We could perhaps do a combination of decreasing (maybe less drastically) minFee and increasing whatever this draftSize is.  I have noticed that with your change to 11000 , we can have maxMinFeeA  to be 100 and the number of discarded tests is still small.

Reply:
Thank you! :slightly_smiling_face: